pom.xml

```diff
@@ -232,6 +232,19 @@
         <artifactId>jaxb</artifactId>
         <version>2.3.0.1</version>
     </dependency>
+    <dependency>
+      <groupId>org.testcontainers</groupId>
+      <artifactId>testcontainers</artifactId>
+      <version>1.10.2</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-test</artifactId>
+      <version>5.0.0.RELEASE</version>
```

Comment:
This pulls in a copy of `spring-core` which is incompatible with Jenkins, making `JCloudsArtifactManagerTest.nonAdmin` fail.

Delete this dep and give `MinioIntegrationTest` package access to `deleteArtifacts` and `deleteStashes` (or `DELETE_ARTIFACTS` and `DELETE_STASHES`).

Reply:
Ok, I'll remove it